docs/ops/sort/NonMaxSuppression_5.md

```diff
@@ -74,6 +74,8 @@ class must not exceed `max_output_boxes_per_class`.
 
 *   **3**: `valid_outputs` - 1D tensor with 1 element of type *T_IND* representing the total number of selected boxes. Optional.
 
+Plugins not supporting dynamic output tensors produce tensor of shape `[min(num_boxes, max_output_boxes_per_class) * num_batches * num_classes, 3]` which is an upper bound for the number of possible selected boxes. Output tensor elements following the really selected boxes are filled with value -1.
```

Comment:
Please, change "Plugins not supporting dynamic output tensors produce tensor of shape..." to "Plugins not supporting dynamic output tensors produce `selected_indices` and `selected_scores` tensors of shape...". I just figured out that there is a third output tensor for which the comment is not applicable.

Reply:
Fixed.